src/ripple/nodestore/impl/Shard.h

```diff
@@ -149,6 +150,14 @@ class Shard final
         removeOnDestroy_ = true;
     }
 
+    bool
+    callForLedgerSQL(
+        std::function<bool(soci::session& session, LedgerIndex index)> const& callback);
```

Comment:
I think the callback parameter is a shard index, not a `LedgerIndex`, in which case the function parameter should be `std::uint32_t`. Ditto for below.

Reply:
Done.

Reply:
This doesn't appear to be done, I'm still seeing `LedgerIndex` as the param.

Reply:
Done.